rpm/libsndfile.spec

```diff
@@ -1,6 +1,6 @@
 Name:       libsndfile
 Summary:    Library for reading and writing sound files
-Version:    1.0.29
+Version:    1.0.31
 Release:    1
 License:    LGPLv2+
 URL:        http://www.mega-nerd.com/libsndfile/
```

Comment:
This seems to be out of date. Apparently they moved [their website](http://libsndfile.github.io/libsndfile/).

Reply:
Thanks, updated.